graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -115,6 +115,17 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
   c10::intrusive_ptr<SampledSubgraph> InSubgraph(
       const torch::Tensor& nodes) const;
 
+  /**
+   * @brief Sample neighbors for a given set of nodes.
```

Comment:
Sample neighboring edges of the given nodes and return the induced subgraph.

Reply:
Fixed